Source/Lib/Codec/EbCodingLoop.c

```diff
@@ -2867,11 +2867,11 @@ void EncodePassPackLcu(
     if ((sequenceControlSetPtr->staticConfig.compressedTenBitFormat == 1))
```

Comment:
@lijing0010 , what did you need to change here? How can we change this TODO comment now?

Reply:
I mean this "//TODO: Jing, need change here late" at line 2866

Reply:
removed